package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngprogress",
-    "version": "1.1.2",
+    "version": "1.1.3",
   "description": "slim, site-wide progressbar for AngularJS",
   "main": "ngProgress.js",
```

Comment:
This pr fixed the `bower.json`, but not `package.json`. With bower being replaced by npm for a lot of devs this is an issue with module resolution.